SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialActorChannel.h

```diff
@@ -179,6 +179,8 @@ class SPATIALGDK_API USpatialActorChannel : public UActorChannel
 	
 	void UpdateEntityACLToNewOwner();
 
+	bool shouldChangeAuthorityIntent();
```

Comment:
This could be a const function.

Reply:
Also "ShouldChangeAuthorityIntent" - capital S

Reply:
Maybe should be a private function?

Reply:
Is this not a private function? I put it with the other private functions in the header

Made function const, I think

Fixed capitalisation

Reply:
Oh, it seems I am a chimp. Yeah it looks private to me :P